src/dmd/backend/divcoeff.d

```diff
@@ -253,6 +255,7 @@ extern (C) bool udiv_coefficients(int N, ullong d, int *pshpre, ullong *pm, int
     return mhighbit;
 }
 
+@trusted
```

Comment:
Why would you need that on a `unittest` ?

Reply:
Because that unittest does unsafe things and triggers the compiler.

Reply:
Note that in all this I am **not** attempting to fix **any** of the unsafe code. I am instead just marking everything as @safe, and then marking as @trusted anything that fails to compile. This is what I've recommended before for anyone wanting to transition to safe-by-default code. It's high time we dogfooded this.